### ISSUE TITLE
[Bug] Templating in tag_values

### ISSUE THREAD

**Issue by davidone** [grafana/grafana#7557]
Grafana version: 4.1.1-1484211277
Datasource: OpenTSDB 2.2
OS: Ubuntu 16.04.1 LTS (xenial)

I have a Dashboard A where I created a 'Text Only' Panel with a Drill Down on Dashboard B using:
Url params: var-foo=bar

In Dashboard B I created:
- a foo variable (Name: foo, Type: Constant, Value: <empty>)
- a host variable (Name: host, Type: Query, Data source; OpenTSDB 2.2, Refresh: On Dashboard Load, Query: tag_values(cpu, host, cluster=$foo))

Drilling down from Dashboard A to Dashboard B I'm expecting a request to OpenTSBD like this:
Request URL: http://localhost/grafana/api/datasources/proxy/2/api/search/lookup?limit=3000&m=cpu%7Bhost%3D*,cluster%3Dbar%7D

Instead I have something like this:
Request URL: http://localhost/grafana/api/datasources/proxy/2/api/search/lookup?limit=3000&m=cpu%7Bhost%3D*,cluster%3Db,foo%3Da,foo%3Dr%7D

Basically, instead of:
/lookup?limit=3000&m=cpu{host=*, cluster=bar}

I have:
/lookup?limit=3000&m=cpu{host=*, cluster=b, foo=a, foo=r}

So I have a red popup with: "Template variables could not be initialized: undefined"

**Comment by torkelo** [comment]
this is a bug in 4.1, fixed in master, https://github.com/grafana/grafana/pull/7208/files
